src/libs/EmojiUtils.js

```diff
@@ -415,7 +415,7 @@ function suggestEmojis(text, lang, limit = CONST.AUTO_COMPLETE_SUGGESTER.MAX_AMO
  * @returns {Number}
  */
 const getPreferredSkinToneIndex = (val) => {
-    if (!_.isNull(val) && Number.isInteger(Number(val))) {
+    if (!_.isNull(val) && _.isUndefined(val) && Number.isInteger(Number(val))) {
```

Comment:
I am having trouble seeing how the two statements could both be true.

`_.isUndefined(val)` -> true

`Number.isInteger(Number(undefined))` -> must be false

Perhaps you meant for this to be `!_.isUndefined(val)`?

Reply:
yes, sorry, that's what i meant to do 👍